oneview_redfish_toolkit/tests/blueprints/test_computer_system_collection.py

```diff
@@ -126,7 +126,7 @@ def test_get_computer_system_collection(self):
         self.assertEqualMockup(computer_system_collection_mockup, result)
 
         self.oneview_client.server_hardware.get_all.assert_called_with(
-            filter="state=ProfileApplied")
+            filter="NOT 'serverProfileUri' = NULL")
```

Comment:
Agree

Reply:
Good, but this will not cause effect, because we are mocking the requests. The filter is applied using the request to Oneview (it is not proccessed in our code) 😞. For it reason I have used the assert by URI of request to oneview: see [here](https://github.com/HewlettPackard/oneview-redfish-toolkit/pull/418/files#diff-a45922826266a8131a38331754ebb8ecR128).

Reply:
You're right, but my idea it's not to test the changed request, but test if the rest of our code can handled a server hardware with this properties the same way it already handles server hardware on "ProfileApplied" state.